packages/components/src/components/text-field/text-field.tsx

```diff
@@ -54,14 +54,20 @@ export class TextField {
   @Prop() name?: string = '';
   /** Input label */
   @Prop() label: string = '';
-  /** (optional) Input size */
-  @Prop() size?: string = '';
+  /** @deprecated - css overwrite should replace size */
+  @Prop() size?: string;
   /** (optional) Input helper text */
   @Prop() helperText?: string = '';
   /** @deprecated - invalid should replace status */
   @Prop() status?: string = '';
   /** (optional) Input status */
   @Prop() invalid?: boolean = false;
+  /** (optional) Input status */
+  @Prop() info?: boolean = true;
+  /** (optional) Input status */
+  @Prop() warning?: boolean = false;
+  /** (optional) Input status */
+  @Prop() success?: boolean = false;
```

Comment:
Since these are mutually exclusive I don't think this will work, let's talk about it